skt/kernelbuilder.py

```diff
@@ -234,11 +249,73 @@ def assemble_make_options(self):
         """Assemble all of the make options into a list."""
         kernel_build_argv = (
             self.make_argv_base
-            + self.targz_pkg_argv
+            + self.compile_args
             + self.extra_make_args
         )
         return kernel_build_argv
 
+    def find_rpm(self):
```

Comment:
Find rpm returns an RPM repository :thinking:. Maybe `get_rpm_repo`? Then the docstring could be more explicit I guess as well.

Reply:
I'll separate the logic there.